Makefile

```diff
@@ -4,20 +4,16 @@ FEDORA_VERSION ?= f32
 default:  help
 
 LIB_T  := $(PREFIX)/var/lib/systemd-swap
-ETCD_T := $(PREFIX)/etc/systemd/swap.conf.d/
-
 BIN_T  := $(PREFIX)/usr/bin/systemd-swap
 SVC_T  := $(PREFIX)/lib/systemd/system/systemd-swap.service
+DFL_T  := $(PREFIX)/usr/share/systemd-swap/swap-default.conf
```

Comment:
This should just say `/usr/share/systemd-swap/swap.conf` since that is what you wrote in the `swap.conf` file (or you should change that file to refer to `swap-default.conf`

Reply:
I guess `swap-default.conf` is a little less confusing when dealing with packaging.